### DISCUSSION TITLE
I Think I Have a More Recent Driver. How Do I Get it to You?

### DISCUSSION THREAD

**Discussion by LycanDarko666** [morrownr/8814au#108] [General]
I have driver date 4-30-2020 version 1030.40.128.2019. Is this what you are looking for?

**Comment by LycanDarko666** [comment]
I'm not really sure how else to get a hold of you.

**Comment by dubhater** [comment]
What is the contents of ``rtw_version.h`` in your driver?

To me ``1030.40.128.2019`` looks like the version of a Windows driver. :)

**Comment by morrownr** [comment]
> What is the contents of rtw_version.h in your driver?

```
#define DRIVERVERSION	"v5.8.5.1_35583.20191029"
```

**Comment by dubhater** [comment]
@morrownr I know! Your ``rtw_version.h`` is right here in this repo. I was asking @LycanDarko666 for theirs, to see if it really is newer.

**Reply by LycanDarko666** [reply]
Ahh, mine is the Windows version. Damn. I was hoping that wouldn't matter. We all bought what we thought was the best model and the thing has barely functioned in comparison to the models from 15 years ago lol.
